docs/image.rst

```diff
@@ -39,8 +50,15 @@ Classes
 
     The other form creates an empty image with ``width`` columns and
     ``height`` rows. Optionally ``buffer`` can be an array of
-    ``width``×``height`` integers in range 0-9 to initialize the image.
-
+    ``width``×``height`` integers in range 0-9 to initialize the image:
+   
+        Image(2, 2, b'\x08\x08\x08\x08')
+	
+    Will display a 2 x 2 pixel image at full brightness.
+    
+    .. note::
+    
+    Keyword arguments cannot be passed to ``buffer``.
```

Comment:
This line needs to be indented to be formatted as a note.

Reply:
done in https://github.com/bbcmicrobit/micropython/pull/582/commits/24b83b74c2cf1bddd8b9c9dcbcef9096979dd24f